packages/suite-desktop-core/src/modules/auto-updater.ts

```diff
@@ -236,8 +242,27 @@ export const init: Module = ({ mainWindow, store }) => {
         logger.info(SERVICE_NAME, `New feed url: ${feedURL}`);
     });
 
+    ipcMain.on('update/set-automatic-update-enabled', (_, value = true) => {
+        logger.info(SERVICE_NAME, `set-automatic-update-enabled: ${value ? 'true' : 'false'}`);
+
+        mainWindow.webContents.send('update/set-automatic-update-enabled', value);
+        const settings = store.getUpdateSettings();
+        store.setUpdateSettings({ ...settings, isAutomaticUpdateEnabled: value });
+        isAutomaticUpdateEnabled = value;
+
```

Comment:
[ ] Todo: trigger download right away when `isAutomaticUpdateEnabled` is set to `true`